tensorflow/core/kernels/sparse/mat_mul_op.cc

```diff
@@ -36,6 +35,7 @@ limitations under the License.
 #include "tensorflow/core/kernels/transpose_functor.h"
 #include "tensorflow/core/lib/gtl/inlined_vector.h"
 #include "tensorflow/core/platform/threadpool.h"
+#include "unsupported/Eigen/CXX11/Tensor"  // from @eigen_archive
 
 #if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
 #include "tensorflow/core/util/cuda_sparse.h"
```

Comment:
Please help add `#include "third_party/gpus/cuda/include/cuda.h"` for the `CUDA_VERSION` macro used in this file. (The line should be below the `gpu_solvers.h` include line because it should be in alphabetical order.)

Reply:
Sure thing, done!